Sources/Extensions/Shared/ColorExtensions.swift

```diff
@@ -31,9 +31,9 @@ public extension Color {
 	
 	/// SwifterSwift: RGB components for a Color (between 0 and 255).
 	///
-	///		UIColor.red.rgbComponents.red -> 255
-	///		NSColor.green.rgbComponents.green -> 255
-	///		UIColor.blue.rgbComponents.blue -> 255
+	///		Color.red.rgbComponents.red -> 255
```

Comment:
I think the issue states that we should not fix the names in the example blocks. Only in the actual documentation